.github/workflows/cypress-test.yml

```diff
@@ -12,92 +12,30 @@ env:
 
 jobs:
   tests:
-    name: Run aggregation view cypress test
+    name: Run Cypress tests
     runs-on: ubuntu-latest
     steps:
-      - name: Download OpenSearch Core
-        run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/3.0.0/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-3.0.0-linux-x64.tar.gz
-          tar -xzf opensearch-*.tar.gz
-          rm -f opensearch-*.tar.gz
-          
-      - name: Download OpenSearch Security Plugin
-        run: wget -O opensearch-security.zip https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/3.0.0/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-security-3.0.0.0.zip
-        
-
-      - name: Run OpenSearch with plugin
-        run: |
-          cat > os-ep.sh <<EOF
-          yes | opensearch-plugin install file:///docker-host/security-plugin.zip
-          chmod +x plugins/opensearch-security/tools/install_demo_configuration.sh
-          yes | plugins/opensearch-security/tools/install_demo_configuration.sh
-          echo "plugins.security.unsupported.restapi.allow_securityconfig_modification: true" >> /opensearch/config/opensearch.yml
-          chown 1001:1001 -R /opensearch
-          su -c "/opensearch/bin/opensearch" -s /bin/bash opensearch
-          EOF
-          docker build -t opensearch-test:latest -f- . <<EOF
-          FROM ubuntu:latest
-          COPY --chown=1001:1001 os-ep.sh /docker-host/
-          COPY --chown=1001:1001 opensearch-security.zip /docker-host/security-plugin.zip
-          COPY --chown=1001:1001 opensearch* /opensearch/
-          RUN chmod +x /docker-host/os-ep.sh
-          RUN useradd -u 1001 -s /sbin/nologin opensearch
-          ENV PATH="/opensearch/bin:${PATH}"
-          WORKDIR /opensearch/
-          ENTRYPOINT /docker-host/os-ep.sh
-          EOF
-          docker run -d -p 9200:9200 -p 9600:9600 -i opensearch-test:latest
-
-      - name: Checkout OpenSearch Dashboard
+      - name: Checkout Branch
         uses: actions/checkout@v2
-        with:
-          path: OpenSearch-Dashboards
-          repository: opensearch-project/OpenSearch-Dashboards
-          ref: 'main'
-          fetch-depth: 0
-      
-      - name: Create plugins dir
-        run: |
-          cd ./OpenSearch-Dashboards
-          mkdir -p plugins
       
-      - name: Checkout OpenSearch Dashboard Security plugin
-        uses: actions/checkout@v2
+      - name: Run OpenSearch with Security Plugin
+        uses: ./.github/actions/install-opensearch
```

Comment:
Can you use the versions that @scrawfor99 has been working on rather than copy/pasting the document into dashbaords?  The syntax would looking something like `uses: {USER/ORG}/{REPO}/.github/actions/`.

If not was there parts that didn't work in this repo?

Reply:
You should be able to run it with `uses: .security/.github/actions/install-opensearch` or something similar once my changes are merged. The one thing is that you may need to to modify the ending behavior should you go that route. For instance I see that you install the security plugin and then do likewise with dashboards, you may have to add an extra input/parameters to install them back to back should running the action twice in a row not prove effective.

Reply:
Got it. Thanks for the info! @scrawfor99 @peternied

Reply:
@RyanL1997 Haven't seen movement here, anything we can help with?

Reply:
Here are the two things I'm working on before we continue working on this PR:
1. I'm fixing the SAML integ test right now. Since the firefox driver is not included in the latest update of selenium yet [1], I am creating a PR to fix that. 
2. Another blocker of this PR is that I have tried to applied the reusable action for integ test, however, the SAML integ test will fail for unknown reason [2]. (it only shows that it cannot locate UI elements). But, if I used the previous set up it will be run [3].

Reference log:
[1]: https://github.com/actions/runner-images/issues/6399#issuecomment-1328761049
[2]: https://github.com/RyanL1997/security-dashboards-plugin/actions/runs/3446947726/jobs/5752434190#step:6:144
[3]: https://github.com/RyanL1997/security-dashboards-plugin/actions/runs/3441246238/jobs/5740580432

Reply:
Is it able to find any of the buttons in the new version? If not: It may require you to run the command in the background with `&`; looking at your changes, I am not sure where exactly you would need to specify this since it seems you are reusing the previous javascript file. But there is a chance that the new formatting no longer runs the OpenSearch or OpenSearch-dashboards in the background like expected and then the later step cannot find any of the elements because it is not persisting. 

If you can find some of the elements, then you know that things are persisting properly but something is likely wrong with the steps that create the buttons you need. For that, I have less of an idea but perhaps the buttons are trying to be created on a different instance of the dashboard?

Reply:
@scrawfor99 Thanks for the info! I will try that out and let you know. However, before I do that I need to fix some of the issues we are having right now of integration test. I just created this [issue](https://github.com/opensearch-project/security-dashboards-plugin/issues/1228) to track the tasks need to be done first.